tools/Modules/RFC/RFC.psm1

```diff
@@ -0,0 +1,369 @@
+$script:repoBase = "https://api.github.com/repos"
+
+<#
+.SYNOPSIS
+Get the highest RFC number which is actually part of the repo
+
+.DESCRIPTION
+Get the highest RFC number which is actually part of the repo
+
+.EXAMPLE
+PS> get-maxrfcnumber
+36
+
+.NOTES
+
+#>
+
+function Get-MaxRFCNumber {
+    $repoOwner = "PowerShell"
+    $repoName = "PowerShell-RFC"
+    $repoFiles = Get-RepoFileList -repoOwner $repoOwner -repoName $repoName
+    $pattern = "RFC(?<number>\d\d\d\d)-"
+    $numbers = $repoFiles.Where({$_.path -cmatch $pattern}).Foreach({if ( $_ -match $pattern ) { [int]($matches.number)}}) | Sort-Object
+    $numbers[-1]
+}
+
+<#
+.SYNOPSIS
+Get the repo file information for the highest RFC
+
+.DESCRIPTION
+Get the repo file information for the highest RFC
+
+.EXAMPLE
+PS> get-maxrfc
+
+size  path                                            file_url
+----  ----                                            --------
+10393 2-Draft-Accepted/RFC0036-AdditionalTelemetry.md https://github.com/PowerShell/PowerShell-RFC/blob/master/2-Draft-Accepted/RFC0036-AdditionalTelemetry.md
+
+.NOTES
+General notes
+#>
+function Get-MaxRFC {
+    $maxNumber = Get-MaxRFCNumber
+    $rfcPattern = "RFC{0:0000}" -f $maxNumber
+    $RFC = (Get-RepoFileList -repoOwner PowerShell -repoName PowerShell-RFC).Where({$_.path -cmatch $rfcPattern})
+    $RFC | Add-Member -TypeName RepoFile -PassThru -MemberType NoteProperty -Name file_url -Value ("https://github.com/PowerShell/PowerShell-RFC/blob/master/" + $RFC.path)
+}
+
+# get the RFCs referenced in PRs
+<#
+.SYNOPSIS
+Retrieve the RFC numbers which are part of pull requests
+
+.DESCRIPTION
+Retrieve the RFC numbers which are part of pull requests
+
+.PARAMETER State
+The state of the PR, open, closed or all
+
+.EXAMPLE
+An example
+
+.NOTES
+General notes
+#>
+
+function Get-PullRFCNumber {
+    param (
+        [ValidateSet("open","closed","all")]
+        [Parameter()]$State = "all"
+    )
+
+    $rfcs = [System.Collections.ArrayList]::new()
+    $page = 1
+    do {
+        $st = Invoke-RestMethod "${repoBase}/PowerShell/PowerShell-RFC/pulls?state=${State}&per_page=100&page=${page}"
```

Comment:
You can use `-FollowRelLink` to ensure you get all of them if there's multiple pages.  Just keep it mind you'll get back a collection of collections so just unroll them in the pipeline.

Reply:
done